app-boilerplate/frontend/.env

```diff
@@ -5,7 +5,7 @@ REACT_APP_LANG="en"
 # This is useful to test a pod provider in a local dev environment
 REACT_APP_POD_PROVIDER_DOMAIN_NAME=
 
-REACT_APP_BACKEND_DOMAIN_NAME=localhost:3001
+REACT_APP_BACKEND_CLIENT_ID=http://localhost:3001/actors/app
```

Comment:
The client_id is where the Client ID document is, and that's on the backend.
On the same URL, you have the `as:Application` and `interop:Application`.
But I agree there is a possible confusion. Should we have two clients ID for backend and frontend ? That wouldn't make much sense since it's the same app requesting the same rights. I haven't seen anything on Solid for this case. Maybe something to discuss with them...

Reply:
I want to add that, for production app, I plan to use the same domain name. Traefik will redirect all `/api` and `/.well-known` calls to the middleware. It will thus be an hybrid application, much like [NextJS API routes](https://nextjs.org/docs/pages/building-your-application/routing/api-routes).